### PR TITLE
Allow filtering folders and files by name

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

**Review by zachmullen — APPROVED** [review]
LGTM once the unused import is removed

**Comment by danlamanna** [comment]
done (no merge permission).

**Comment by zachmullen** [comment]
@brianhelba can we disable the "Restrict who can push to matching branches" checkbox on master, or was that on intentionally?

**Pull request merged**
